### PR TITLE
Issue/653/admin user management

### PR DESCRIPTION
**Pull request by fzemi**
Closes HAR-17

### CODE REVIEW

File: server/endpoints/v1/users/methods/update-user.js [hunk 1]
```diff
@@ -1,7 +1,198 @@
...
 const { ObjectId } = require('mongodb');
+const { AppError, errorCodes } = require('../../../../libs/errors');
+const getUserForAdminAggregation = require('../../../../aggregations/get-user-for-admin');
+const { generateUniqueKey } = require('../../../../libs/utils');
+
+async function updateUser (userId, userObject) {
+
+  const user = await Users.get({ _id: ObjectId(userId) }, {
+    aggregationPipeline: getUserForAdminAggregation,
+  });
```
Comment (HenoJurek) [inline server/endpoints/v1/users/methods/update-user.js:14]: Do we have any validation for `userId`, does user with this id exist?

File: server/endpoints/v1/users/methods/update-user.js [hunk 2]
```diff
@@ -1,7 +1,198 @@
...
+    const userEvents = {};
+    const { role: actualRole } = await UsersEvents.get({ _id: ObjectId(userEventsData[idx]) });
+
+    if (actualRole === 'teamLeader' || actualRole === 'creator') {
+      throw new AppError(errorCodes.CANNOT_UPDATE_USEREVENTS_ROLE, {
+        httpStatus: 400,
+      });
+    }
+
+    if (received.nickname) {
```
Comment (HenoJurek) [inline server/endpoints/v1/users/methods/update-user.js:78]: This can be write with destructuring(all variables from `received`)

**Review by HenoJurek — CHANGES_REQUESTED** [review]

File: server/endpoints/v1/users/methods/update-user.js (hunk 1 above)
Comment (fzemi) [reply server/endpoints/v1/users/methods/update-user.js:14]: Done

File: server/endpoints/v1/users/methods/update-user.js (hunk 2 above)
Comment (fzemi) [reply server/endpoints/v1/users/methods/update-user.js]: Done

**Review by HenoJurek — APPROVED** [review]

**Pull request merged**